README.md

```diff
@@ -62,15 +63,18 @@ All wrapper layers are preconfigured to use Splunk Observability Cloud as the tr
     OTEL_EXPORTER_JAEGER_ENDPOINT: <endpoint URL>
     ```
    
+   > **_NOTE:_**  Currently Splunk Observability Cloud supports `jaeger/thrift` and `OTLP` with `http/protobuf` protocols. In order to send traces directly to ingest, set following environment variable in your Lambda function code:
```

Comment:
I think it'd probably be better to use `Jaeger Thrift` -- `jaeger-thrift` looks like it could be confused with the OTLP protocol.